source/reference/api/start.txt

```diff
@@ -22,11 +22,41 @@ Starts the synchronization between a source and destination cluster.
 Requirements
 ------------
 
+State
+~~~~~
+
 To use the ``start`` endpoint, ``mongosync`` must be in the ``IDLE``
 state.
 
+User Write Blocking
+~~~~~~~~~~~~~~~~~~~
+
 .. include:: /includes/fact-write-blocking-requirement.rst
+
+To set a custom role for the ``mongosync`` user:
+ 
+#. To create a custom role, use the :dbcommand:`createRole` command:
+
+   .. code-block:: javascript
+   
+      db.adminCommand( {
+         createRole: "reverseSync",
+         privileges: [ {
+             resource: { db: "", collection: "" },
+             actions: [ "setUserWriteBlockMode", "bypassWriteBlockingMode" ]
+         } ],
+         roles: []
+      } )
  
+#. To grant the custom role to the ``mongosync`` user, use the :dbcommand:`grantRolesToUser` command:
+
+   .. code-block:: javascript
+
+      db.adminCommand( {
+         grantRolesToUser: "mongosync",
+         roles: [ { role: "reverseSync", db: "admin" } ]
+      } )
+
```

Comment:
This section is super helpful! A suggestion (don't feel super strongly about this, so up to you): maybe we could add a line at the end of this section saying something along the lines of: "Make sure to use this newly created `mongosync` user in the connection string arguments in starting up the `mongosync` binary."

This will be helpful to tie back the user and role creation to how we use them in mongosync.

Reply:
Fixed